src/lib/logger.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-module.exports = require('./httpd').log;
```

Comment:
I think we'll still need this. Take `lib/metrics.js` for example. It's still expecting this file to export something called `log`. I would move this line below where `addStream()` is called.

Reply:
Agreed, and resolved